srcpkgs/Carla/template

```diff
@@ -0,0 +1,48 @@
+# Template file for 'Carla'
+pkgname=Carla
+version=1.9.8
+revision=1
+build_style=gnu-makefile
+hostmakedepends="pkg-config"
+makedepends="
+ python3-PyQt5-devel-tools python3-PyQt5 libmagic file-devel liblo-devel
+ alsa-lib-devel pulseaudio-devel libX11-devel gtk+3-devel gtk+-devel qt5-devel
+ qt-devel fluidsynth-devel fftw-devel mxml-devel zlib-devel python3-rdflib"
+depends="python3 python3-PyQt5 python3-PyQt5-svg"
+short_desc="Audio plugin host"
+maintainer="nutcase84 <nutcase84@protonmail.com>"
+license="GPL-2.0-or-later"
+homepage="http://kxstudio.linuxaudio.org/Applications:Carla"
+distfiles="https://github.com/falkTX/Carla/archive/${version}.tar.gz"
+checksum=966647c508bf92eafb03d0aaf687acb399c687782461154fad40d64745786e67
+
+case $XBPS_TARGET_MACHINE in
+	x86_64* | i686*);;
+	*) make_build_args+=" NOOPT=true"
+	   make_install_args+=" NOOPT=true";;
+esac
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" Carla-devel"
+fi
+
+pre_build() {
+	if [ "$CROSS_BUILD" ]; then
+		sed -i -e 's#\./carla-lv2-export#/usr/bin/carla-lv2-export#g' \
+			source/plugin/Makefile
+	fi
+
+	make ${make_build_args} features
+}
+
+post_install() {
+	mv ${DESTDIR}/usr/lib/python3{,.5}
```

Comment:
wth is this? Please fix.

Reply:
I'm not entirely sure about why this is, but I think Carla writes to /usr/lib/python3 but the hook expects a specific python version to be specified. Hook 03-rewrite-python-shebang will fail without this hack.